CBDatabase/DB/Database.swift

```diff
@@ -301,8 +301,6 @@ public final class Database {
                     "\(refreshedObjects.count) refresh(s)",
                 ]
 
-                print("[databases]: [\(T.entityName)] \(pieces.joined(separator: ", "))")
```

Comment:
Could we leave this in for `DEBUG` builds? It's pretty helpful

Reply:
Yeah, we should keep this. Otherwise, we don't know what's going on during app syncs. If we need a better way to handle log levels across the app, we should maybe do that.

Reply:
@AndrewGold @hishbase - updated to use `os_log`, which will print to the Xcode console same as before but will be easier to filter when using the `Console` app